CODEOWNERS

```diff
@@ -283,7 +283,7 @@ extensions/filters/http/oauth2 @derekargueta @snowp
 # Header to metadata
 /*/extensions/filters/http/header_to_metadata @zuercher @JuniorHsu
 # zookeeper
-/*/extensions/filters/network/zookeeper_proxy @snowp @JuniorHsu
```

Comment:
Could we add @Winbobob as he's the most active contributor on this component?

https://github.com/envoyproxy/envoy/commits/main/source/extensions/filters/network/zookeeper_proxy